types/component/AnonymousChat/BaseChatItem.types.ts

```diff
@@ -22,7 +22,9 @@ export interface BaseChatItemProps {
 
 export interface ChatItemMyTextProps extends BaseChatItemProps {
   status?: ChatStatus;
-  chatType?: string;
+  chatType: 'SIGNED' | 'ANONYMOUS';
+  messageType: string;
+  data: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `chatType` property has been changed from optional to required and its type has been restricted to `'SIGNED' | 'ANONYMOUS'`. This could potentially break existing code if it's not updated accordingly. Also, the `messageType` and `data` properties have been added without any type definitions which can lead to potential type errors in the future. It would be better to provide specific types for these properties.

```diff
-  chatType: 'SIGNED' | 'ANONYMOUS';
+  chatType?: 'SIGNED' | 'ANONYMOUS';
-  messageType: string;
+  messageType?: MessageType; // Assuming MessageType is a defined type
-  data: any;
+  data?: DataType; // Assuming DataType is a defined type
```
Please ensure that you update all places where this interface is used to reflect these changes and avoid potential runtime errors.


<!-- This is an auto-generated comment by OSS CodeRabbit -->